update.sh

```diff
@@ -8,44 +8,52 @@
 
 set -e
 
+# Check whether there is a newer version of this script
+cd /usr/share/rhino/rhino-update
+if git pull; then
+    chmod +x /usr/share/rhino/rhino-update/update.sh
+    exec /usr/bin/rhino-update
+    exit $?
+fi
+
 # Check to see whether the "configuration update", released in 2022.04.19 has been applied.
-if [[ ! -f "$HOME/.rhino/updates/configuration" ]]; then
-  mkdir -p ~/.rhino/{config,updates}
-  echo "alias rhino-config='mkdir ~/.rhino/config/config-script && git clone https://github.com/rollingrhinoremix/rhino-config ~/.rhino/config/config-script/ && python3 ~/.rhino/config/config-script/config.py && rm -rf ~/.rhino/config/config-script'" >> ~/.bashrc
-  : > "$HOME/.rhino/updates/configuration"
+if [[ ! -f "/usr/share/rhino/updates/configuration" ]]; then
+  mkdir -p /usr/share/rhino/{config,updates}
+  echo "alias rhino-config='mkdir /usr/share/rhino/config/config-script && git clone https://github.com/rollingrhinoremix/rhino-config /usr/share/rhino/config/config-script/ && python3 /usr/share/rhino/config/config-script/config.py && rm -rf /usr/share/rhino/config/config-script'" >> ~/.bashrc
+  : > "/usr/share/rhino/updates/configuration"
```

Comment:
This seems un-necessary as it's creating aliases and was for an old version of Rolling Rhino Remix. I believe this can be removed.